Cargo.lock

```diff
@@ -13,9 +13,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.56"
```

Comment:
Not happy about the changes in this file. It seems you did a `cargo update`. So now here are a lot of dependency updates that are not related to this PR and should be in their own dedicated commit.

Reply:
The cargo update is related to this PR (needed for the new image crate version) and are indeed in a separa commit.

Reply:
Hmm, not sure if I'm holding it wrong, but this Cargo.lock diff is all I need (Remove the .zip extension). It has far fewer version bumps for unrelated crates.

[Cargo-lock.diff.zip](https://github.com/melix99/telegrand/files/8582248/Cargo-lock.diff.zip)

Reply:
Oh, you mean that I updated all dependencies instead of just the image one. Well, fair.